plugins/brjs-plugins/src/main/java/org/bladerunnerjs/plugin/require/AliasDataRequirePlugin.java

```diff
@@ -0,0 +1,40 @@
+package org.bladerunnerjs.plugin.require;
+
+import org.bladerunnerjs.api.Asset;
+import org.bladerunnerjs.api.BRJS;
+import org.bladerunnerjs.api.BundlableNode;
+import org.bladerunnerjs.api.model.exception.RequirePathException;
+import org.bladerunnerjs.api.plugin.RequirePlugin;
+import org.bladerunnerjs.api.plugin.base.AbstractRequirePlugin;
+import org.bladerunnerjs.plugin.plugins.require.AliasDataSourceModule;
+
+
+public class AliasDataRequirePlugin extends AbstractRequirePlugin implements RequirePlugin
+{
+	
+	private RequirePlugin defaultRequirePlugin;
+
+	@Override
+	public void setBRJS(BRJS brjs)
+	{
+		this.defaultRequirePlugin = brjs.plugins().requirePlugin("default");
+	}
+
+	@Override
+	public String getPluginName()
+	{
+		return "alias";
+	}
+
+	@Override
+	public Asset getAsset(BundlableNode bundlableNode, String requirePathSuffix) throws RequirePathException
```

Comment:
This method seems to be the only non-test method of any consequence, yet it doesn't seem to seem to do very much, so that I'm struggling to reconcile this method with the claimed commit message &mdash; `adding an aliasDataPlugin that returns alias data for the BundlableNode`.

Can you provide some more detail?